File: CoreJava/Player/src/com/techlabs/player/Player.java
package com.techlabs.player;

public class Player {
	private int id;
	private String name;
	private int age;

	public Player(int id, String name) {
		this(id, name, 18);
	}

	public Player(int id, String name, int age) {
		this.id = id;
		this.name = name;
		this.age = age;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public Player whoIsElder(Player player) {
		return this.getAge() > player.getAge() ? this : player;
	}

	@Override
	public String toString() {
		String result = " Id\t:" + getId() + "\n Name\t:" + getName()
				+ " \n Age\t:" + getAge() + " \n Hash\t:" + super.toString();
		return result;
	}

	// @Override
	public boolean equals(Object obj) {
		return this.getId() == ((Player) obj).getId();
	}
}

File: DotNET/C#/PlayerApp/PlayerApp/Player.cs
using System;

namespace PlayerApp
{
    class Player
    {
        private int _id;
        private String _name;
        private int _age;

        public Player(int id, String name)
        {
            _id = id;
            _name = name;
        }

        public Player(int id, String name, int age)
        {
            _id = id;
            _name = name;
            _age = age;
        }

        public int Id
        {
            get
            {
                return _id;
            }
            set
            {
                _id = value;
            }
        }

        public String Name
        {
            get
            {
                return _name;
            }
            set
            {
                _name = value;
            }
        }

        public int Age
        {
            get
            {
                return _age;
            }
            set
            {
                _age = value;
            }
        }

        public override string ToString()
        {
            String result = " Id\t:" + Id + "\n Name\t:" + Name
                + " \n Age\t:" + Age;
            return result;
        }

        public Player whoIsElder(Player player)
        {
            return Age > player.Age ? this : player;
        }

        public override bool Equals(object obj)
        {
            return Id.Equals(obj);
        }

    }
}
